modules/coreboot

```diff
@@ -110,11 +107,12 @@ coreboot_dir := $($(coreboot_module)_dir)
 coreboot_base_dir := $($(coreboot_module)_base_dir)
 
 $(coreboot_module)_depends += $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
-$(coreboot_module)_depends += $(if $(CONFIG_NITROKEY_BLOBS), nitrokey-blobs)
 
 # coreboot builds are specialized on a per-target basis.
 # The builds are done in a per-target subdirectory
 CONFIG_COREBOOT_CONFIG ?= config/coreboot-$(BOARD).config
+CONFIG_COREBOOT_LOCALVERSION ?= "$(BRAND_NAME)-$(HEADS_GIT_VERSION)"
+CONFIG_COREBOOT_SMBIOS_PRODUCT_NAME ?= $(BOARD)
```

Comment:
```suggestion

# The BIOS version, manufacturer, and product name may be overridden by site-local/config
# for branded devices
CONFIG_COREBOOT_LOCALVERSION ?= "$(BRAND_NAME)-$(HEADS_GIT_VERSION)"
CONFIG_COREBOOT_SMBIOS_PRODUCT_NAME ?= $(BOARD)
CONFIG_COREBOOT_SMBIOS_MANUFACTURER ?=
```

I see what the objective is for overriding these in site-local/config for rebranded devices.  I'd suggest documenting the intent here so it's clear, folks like me will not ask about it over and over :wink: 

Maybe in the future we consider defaulting the board name to "not overridden" instead of the Heads board name, but that should probably be addressed separately.

Reply:
IIUC we have competing schemes https://github.com/Dasharo/dasharo-issues/issues/762

Please note that Dasharo cannot change the naming scheme arbitrarily because some customers and systems rely on that. Of course that doesn't mean our naming scheme is correct, but I only can vote for the scheme that generates more value e.g. saving developers time or serves better for users. I cannot resign from Dasharo branding despite the fact I would like to endorse Heads. So what, I think we should discuss is Heads and branding maybe even trademarks, what may need dedicated issue.

Going further Dasharo Supporting Pattern Agreement is the only prototype legal document which tries to establish relation between two entities (in this case 3mdeb owner of Dasharo and @tlaurion maintainer of Heads). Finally lets ask ourselves what is the legal status of heads branding?

P.S. no excuses,  but I honestly would love to have more time to address those issues in correct way